django/core/management/base.py

```diff
@@ -277,6 +279,16 @@ def create_parser(self, prog_name, subcommand, **kwargs):
         Create and return the ``ArgumentParser`` which will be used to
         parse the arguments to this command.
         """
+
+        def add_base_argument(parser: CommandParser, name, *args, **kwargs):
+            """
+            wrapper for CommandParser.add_argument
+            suppresses the help text according to BaseCommand.suppressed_base_arguments
+            """
+            if name in self.suppressed_base_arguments:
```

Comment:
I would change this so each of the positional arguments is checked against `self.suppressed_base_arguments`. Then e.g. `--verbosity` doesn't have to be listed first when both `-v` and `--verbosity` are provided. Also, if this is done, then `name` doesn't need to be distinguished / given a name in the argument list.

Reply:
Thanks @cjerdonek . 
I watched for positional arguments to not conflict with it as per documentation:
[argparse.add_argument() - name-or-flags](https://docs.python.org/3/library/argparse.html#name-or-flags)

Shall I add a test for positional argument conflict:
`add_base_argument(parser,'foo', 'version')` should not suppress the help for `--version`.

Reply:
Added positional support as suggested in other comment.